src/Ilios/CoreBundle/EventListener/UpdateSessionTimestamp.php

```diff
@@ -44,7 +44,7 @@ public function onFlush(OnFlushEventArgs $eventArgs)
         foreach ($entities as $entity) {
             switch (get_class($entity)) {
                 case 'Ilios\CoreBundle\Entity\IlmSessionFacet':
-                    $sessions = array_merge($sessions, $entity->getSessions()->toArray());
+                    $sessions[] = $entity->getSession();
```

Comment:
I think this is returning null which exposes a bug further down on line 55 where it attempts to do doctrine stuff on whatever is passed without checking.  I think the best thing to do would be to filter the $sessions array after line 51 to ensure that all the values are a SessionInterface Entity and not something else.